src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java

```diff
@@ -310,14 +310,16 @@ public void xpendeing() {
     
     List<StreamPendingEntry> pendingRange = jedis.xpending("xpendeing-stream", "xpendeing-group", null, null, 3, "xpendeing-consumer");
     assertEquals(1, pendingRange.size());
-    
+
     try {
       Thread.sleep(1000);
     } catch (InterruptedException e) {
       e.printStackTrace();
     }
     
     jedis.xclaim("xpendeing-stream", "xpendeing-group", "xpendeing-consumer2", 500, 0, 0, false, pendingRange.get(0).getID());
+
+    jedis.xclaimIDs("xpendeing-stream", "xpendeing-group", "xpendeing-consumer2", 500, 0, 0, false, pendingRange.get(0).getID());
```

Comment:
maybe this can be improved to check the output of the command?

Reply:
Thanks. 
I think no one seems to be maintaining this project.Only two of us submited PR recently.＋﹏＋

Reply:
You have to remember that people do it in their free time :)